<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <rosparam file="$(find pcl_proc)/config/common_costmap_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find pcl_proc)/config/common_costmap_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find pcl_proc)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find pcl_proc)/config/global_costmap_params.yaml" command="load" />
    </node>
</launch>
